src/components/ProductVersionReleasesList/ProductVersionReleasesList.tsx

```diff
@@ -0,0 +1,112 @@
+  filterAttributes: {
+    version: {
+      id: 'version',
+      title: 'Name',
+      operator: '=like=',
+    },
+    supportLevel: {
+      id: 'supportLevel',
+      title: 'Support Level',
+      filterValues: ['UNRELEASED', 'EARLYACCESS', 'SUPPORTED', 'EXTENDED_SUPPORT', 'EOL'],
+      operator: '==',
+    },
+  },
+};
+
+interface IProductVersionReleasesListProps {
+  serviceContainerProductReleases: IServiceContainer;
+  componentId: string;
+}
+
+/**
+ * Component displaying list of Product Releases.
+ *
+ * @param serviceContainerProductReleases - Service Container for Product Releases
+ * @param componentId - Component ID
+ */
+export const ProductVersionReleasesList = ({
+  serviceContainerProductReleases,
+  componentId,
+}: IProductVersionReleasesListProps) => {
+  const { serviceContainerProductVersion } = useServiceContainerProductVersion();
+
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <Filtering filterAttributes={filterAttributes} componentId={componentId} />
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop>
+        <ServiceContainerLoading {...serviceContainerProductReleases} title="Product Releases">
+          <TableComposable isStriped variant="compact">
+            <Thead>
+              <Tr>
+                <Th width={25}>Name</Th>
+                <Th width={20}>Release Date</Th>
+                <Th width={20}>Released In</Th>
+                <Th width={20}>Support Level</Th>
+                <Th />
+              </Tr>
+            </Thead>
+            <Tbody>
+              {serviceContainerProductReleases.data?.content.map((productRelease: ProductRelease, rowIndex: number) => (
+                <Tr key={rowIndex}>
+                  <Td>
+                    <ProductMilestoneReleaseLabel productMilestoneRelease={productRelease} isCurrent={false} />
+                  </Td>
+                  <Td>{productRelease.releaseDate && createDateTime({ date: productRelease.releaseDate }).custom}</Td>
+                  <Td>
+                    {productRelease.productMilestone && (
+                      <ProductMilestoneReleaseLabel
+                        link={`../milestones/${productRelease.productMilestone.id}`}
+                        productMilestoneRelease={productRelease.productMilestone}
+                        isCurrent={
+                          productRelease.productMilestone.id === serviceContainerProductVersion.data?.currentProductMilestone?.id
+                        }
+                      />
+                    )}
+                  </Td>
+                  <Td>{productRelease.supportLevel}</Td>
+                  <Td isActionCell>
+                    <ActionsColumn
+                      items={[
+                        {
+                          title: 'Edit the Release',
+                        },
+                        {
+                          title: 'Delete the Release',
+                        },
+                      ]}
```

Comment:
We can use `Edit Release` and `Delete Release` instead of a longer version, which keeps the same with our angular JS UI